### New game status message.
#
# This published message is sent at the start of a new game.
#

Header  header      # standard ROS header
string  white_name  # white player's name
string  black_name  # black player's name
